iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -481,7 +484,19 @@ private void commitTable(FileIO io, ExecutorService executor, OutputTable output
           .map(x -> x.getJobConf().get(ConfVars.REWRITE_POLICY.varname))
           .orElse(RewritePolicy.DEFAULT.name()));
 
-      commitOverwrite(table, branchName, startTime, filesForCommit, rewritePolicy);
+      Integer compactionSpecId = outputTable.jobContexts.stream()
+          .findAny()
+          .map(x -> x.getJobConf().get(CompactorContext.COMPACTION_SPEC_ID))
```

Comment:
is that partition spec id? name is confusing

Reply:
Fixed to make the name more clear